momentum/package-lock.json

```diff
@@ -34,11 +34,27 @@
         }
       }
     },
+    "@types/commander": {
+      "version": "2.12.2",
+      "resolved": "https://registry.npmjs.org/@types/commander/-/commander-2.12.2.tgz",
+      "integrity": "sha512-0QEFiR8ljcHp9bAbWxecjVRuAMr16ivPiGOw6KFQBVrVd0RQIcM3xKdRisH2EDWgVWujiYtHwhSkSUoAAGzH7Q==",
+      "dev": true,
+      "requires": {
+        "commander": "2.15.1"
+      }
+    },
     "@types/d3": {
       "version": "5.0.0",
       "resolved": "https://registry.npmjs.org/@types/d3/-/d3-5.0.0.tgz",
       "integrity": "sha512-BVfPw7ha+UgsG24v6ymerMY4+pJgQ/6p+hJA4loCeaaqV9snGS/G6ReVaQEn8Himn67dWn/Je9WhRbnDO7MzLw==",
+<<<<<<< HEAD
+<<<<<<< HEAD
+      "dev": true,
+=======
+>>>>>>> renders normal distribution, kinda
+=======
       "dev": true,
+>>>>>>> dotplots WIP. committing before break
```

Comment:
I don't think you rebased properly. The package-lock.json should not have these lines.

Reply:
This may have been me. I'm not sure